src/templates/globals/footer/staticData/footer-links.js

```diff
@@ -0,0 +1,40 @@
+export const footerLinks = [
```

Comment:
why is this file necessary if these items all exist in `importHeaderData.js` ?

Reply:
Oversite on my part, and this makes me see there is a good deal of other footer related code I can clean up as well